how-to-win-friends.md

```diff
@@ -0,0 +1,30 @@
+# Book Report
+
+## Title
+
+how to win friends and influence people.
+
+## Author(s)
+DALE CARNEGIE.
+
+
+## What are the main characters in the story?
+does not apply
+
+## What is the setting of the story? (Where and when does it take place?)
+does not apply
+
+## What are the main events/ideas from the story
+the principles.
+principle 1:Begin with praise and honest appreciation.  
+principle 2:Call attention  to people's mistakes indirectly.  
+principle 3:Talk about your mistakes before criticizing the other person.
+principle 4:Ask questions instead of giving direct orders.
+principle 5:Let the other person save face. 
+principle 6:Praise the slightest improvement and praise every enviorment.Be "hearty in your praise.  
+principle 7:Give the other person a fine reputation to live up to. 
+principle 8:Use encouragement.Make fault easy to correct.
+principle 9:Make the other person happy about doing the thing you suggest.
+ If someone reads the principles then i hope you can apply these in your life!
+## How does the story end?
```

Comment:
There should be a linebreak above this line.

Reply:
is it good now?